packages/react-day-picker/src/types/Formatters.ts

```diff
@@ -22,8 +27,29 @@ export type Formatters = {
   formatWeekdayName: DateFormatter;
 };
 
+/** Represent a map of formatters used to render localized content. */
+export type StringFormatters = {
```

Comment:
What is different from `Formatters` above?

Reply:
Same thing as above - the issue is that the current formatters return a React node instead of a string. Technically speaking, the type information there is wrong as the formatters return strings - not React nodes. It hopefully shouldn't conflict if it were to be changed to a `string` return type, however, I did this just in case.